plugin-api.d.ts

```diff
@@ -202,73 +163,62 @@ interface NotificationOptions {
     action: () => boolean | void
   }
 }
-
-type NotifyDequeueReason = 'timeout' | 'dismiss' | 'action_button_click'
-
+declare type NotifyDequeueReason = 'timeout' | 'dismiss' | 'action_button_click'
```

Comment:
I was curious what "declare type" does and turns out it is a noop https://github.com/Microsoft/TypeScript/issues/1940 so this is safe